capa/render/result_document.py

```diff
@@ -521,6 +544,7 @@ def from_capa(cls, rule: capa.rules.Rule) -> "RuleMetadata":
         return cls(
             name=rule.meta.get("name"),
             namespace=rule.meta.get("namespace"),
+            prevalence=load_rules_prevalence().get(rule.meta.get("name"), "unknown"),
```

Comment:
is the rule prevalence database distributed with capa the library? i think its important that people be able to use capa the library without maintaining this database. so perhaps we want to handle the case of the database not existing here?

Reply:
In case database is not present, all rule matches will have prevalence as `unknown` in the results.
![image](https://github.com/mandiant/capa/assets/81844215/ac9db4bd-30ad-4178-8c01-72b492964ec6)

Reply:
maybe we can provide a warning if no db is found (in case that's not already there) pointing to one and explaining shortly what it does